### PR TITLE
Use relation entities for driving licences

### PR DESCRIPTION
**Pull request by vlaaad**

### CODE REVIEW

File: src/jobtech_taxonomy_database/converters/driving_licence_converter.clj [hunk 1]
```diff
@@ -4,7 +4,7 @@
             [jobtech-taxonomy-database.converters.converter-util :as u]
             [jobtech-taxonomy-database.types :as t]))
 
-(def combination-mapping ;; key is drivers licence ID, value array contains implicit IDs
+(def driver-licence-id->implicit-ids
```
Comment (weipe) [inline src/jobtech_taxonomy_database/converters/driving_licence_converter.clj:7]: This map naming convention is nice.

File: src/jobtech_taxonomy_database/converters/driving_licence_converter.clj [hunk 2]
```diff
@@ -21,24 +21,20 @@
...
-    [concept-with-extra-extras]))
+  (let [implicit-ids (driver-licence-id->implicit-ids drivinglicenceid)]
+    (->> implicit-ids
+         (map #(u/create-relation (u/create-temp-id t/driving-licence %)
+                                  (u/create-temp-id t/driving-licence drivinglicenceid)
+                                  t/broader))
+         (cons (assoc (u/create-concept t/driving-licence term description drivinglicenceid)
+                 :concept.external-standard/driving-licence-code-2013 drivinglicencecode
+                 :concept/sort-order displaysortorder
+                 :concept.external-standard/implicit-driving-licences (map implicit-licence-ref implicit-ids))))))
```
Comment (weipe) [inline src/jobtech_taxonomy_database/converters/driving_licence_converter.clj:38]: Nice combo: ->>, map, cons

**Comment by weipe** [comment]
Looks good to me!

File: src/jobtech_taxonomy_database/converters/driving_licence_converter.clj (hunk 2 above)
Comment (vlaaad) [reply src/jobtech_taxonomy_database/converters/driving_licence_converter.clj:38]: Haha, yes, I used `cons` just so I could keep the nice threading :)

**Pull request merged**